experimental/personnames/src/space_replacement.rs

```diff
@@ -0,0 +1,135 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use icu_locid::Locale;
+
+const DEFAULT_FOREIGN_SPACE_REPLACEMENT: &str = " ";
+
+///
+/// https://www.unicode.org/reports/tr35/tr35-personNames.html#setting-the-spacereplacement
+///
+pub(crate) fn space_replacement<'lt>(
+    formatting_locale: &Locale,
+    person_name_locale: &Locale,
+    foreign_space_replacement: Option<&'lt str>,
+) -> &'lt str {
+    let mut native_space_replacement = DEFAULT_FOREIGN_SPACE_REPLACEMENT;
+    if let Some(script) = formatting_locale.id.script {
+        native_space_replacement = match script.as_str() {
+            "Thai" => " ",
```

Comment:
I think you have an ASCII space in this line; is that intentional?

Reply:
The specs only says "space", but there's no indication of which encoding it should be. (so it's UTF-8 space in source code since rust is utf-8, but it should match US-ASCII space AFAIU)

Reply:
I see. If this string is the same as `DEFAULT_FOREIGN_SPACE_REPLACEMENT`, can you just use that instead of hard-coding a character here?

Reply:
Done !